src/__tests__/hooks.test.js

```diff
@@ -879,21 +870,19 @@ describe.skip('hooks', () => {
       return <div>{state && state.data}</div>;
     };
 
-    // Get the first child since the page is wrapped with the WithTracking component.
-    const page = await mount(<Page />).childAt(0);
-    await page.instance().executeAction();
+    const page = await mount(<Page />);
     // TODO: redundant with previous test perhaps, but here for posterity. Similarly we may need to call executeAction(); directly
-    executeAction();
+    await executeAction();
 
-    expect(page.state().data).toEqual(message);
+    expect(page.text()).toEqual(message);
     expect(dispatchTrackingEvent).toHaveBeenCalledTimes(1);
     expect(dispatchTrackingEvent).toHaveBeenCalledWith({
       label: 'async action',
       status: 'failed',
     });
   });
 
-  it('can access wrapped component by ref', async () => {
+  it.skip('can access wrapped component by ref', async () => {
```

Comment:
This is kind of a weird one. I _think_ that calling the ref's `focus` method programmatically does not result in the element's `onFocus` method being called. In any case, I'm not so sure that this test is really testing anything useful here—in the version for the decorator, it's testing that `track` forwards the ref along like we'd expect, but here the hook doesn't actually interact with the ref in any way.

Reply:
Yup, I think that's right. I'll remove this test.